utils/index.d.ts

```diff
@@ -1118,9 +1118,11 @@ export declare interface ExecuteActivityContext {
      */
     activityTitle?: string;
     /**
+     * Resource or resourceId
+     *
      * Set to show a "Click to view resource" child on success.
      */
-    activityResult?: AppResource;
+    activityResult?: AppResource | string;
```

Comment:
In the future, we want to make this only accept a `string`. But for compatibility we'll take `AppResource` too for now.